README.md

```diff
@@ -0,0 +1,48 @@
+# Prototype Form
+
+The project is a prototype of improved form design based on learnings from user research and usability testing with Austin Animal Center Foster Forms.
+
+## Install Jekyll
```

Comment:
Any chance installing docker is an option? I like the 0-setup local environment, especially for folks who don't dev every day. Docker can be a little hard to deal with though so I understand not wanting to bite that off yet.

Reply:
It does look it might be an option but I'm not sure what direction to go:

https://github.com/Praqma/docker-gh-pages
https://github.com/Starefossen/docker-github-pages

Reply:
The latter one looks pretty good. Would one of the jekyll images work for running locally? https://github.com/envygeeks/jekyll-docker/blob/master/README.md